### DISCUSSION TITLE
Acute training load not updating on Garmin

### DISCUSSION THREAD

**Discussion by Lavogel** [philosowaffle/peloton-to-garmin#508] [TE / TSS / Vo2 / Intensity Minutes]
**Describe the bug**
On Garmin Connect (and therefore my Epix 2) my "Acute Training Load" for rides on my Peloton Bike is not updated anymore.

I'm not sure if this is an issue with P2G, but maybe there is someone here who experienced the same problem before. I have unsuccessfully discussed this with Garmin for months now and they are still investigating the source of the issue.

All other informations of an activity, like heart rate, cadence, power zones, etc. are correctly shown in Connect. The ATL however was degrading for a few weeks and finally hit zero someday. 
When doing any other activity, e.g. riding outdoor or with another indoor bike, which is syncing its data to Garmin via Trainer Road, the ATL is starting to climb again, so I guess it has something to do with the way or content of the data that is transferred to Garmin.

**Installation (please complete the following information):**
I am running the latest docker version of P2G 
 - OS: Synology DSM 7.1.1
 - Method: Docker
 - Version: P2G 3.5.1

**Comment by philosowaffle** [comment]
Ultimately, this is a Garmin problem, and for unknown reasons, its always been flaky for some.  There's a bit more information about this on [past discussions](https://github.com/philosowaffle/peloton-to-garmin/discussions/464).

Most recently [we ran into this problem with the new rowers](https://github.com/philosowaffle/peloton-to-garmin/issues/383). Initially, TS/ATL did not show up on synced row workouts, and then all of a sudden, without any changes from P2G, Garmin started calculating those values.
